lib/windows/loader/components/AppListView.jsx

```diff
@@ -34,12 +34,51 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import React from 'react';
+import React, { PropTypes } from 'react';
+import { Iterable } from 'immutable';
+import AppListItem from './AppListItem';
 
-const AppListView = () => (
-    <div>
-        App list
-    </div>
-);
+function getSortedApps(apps) {
+    return apps.sort((a, b) => {
```

Comment:
Good question. I was not 100% sure, so I wrote a test for it, ref 586ec17. If the apps have the same name, then official ones get priority. 

If the component receives apps that have both the same `name` and `isOfficial` values, then the order between them does not matter that much. This should not happen. But if it does, the test shows that the apps are at least sorted correctly with respect to the other apps in the list.